components/Button.tsx

```diff
@@ -21,34 +21,45 @@
 
 import React, { ReactNode, ReactNodeArray } from 'react';
 import { css } from '@emotion/core';
+import isPropValid from '@emotion/is-prop-valid';
 import styled from '@emotion/styled';
 
-import defaultTheme from './theme';
+import { DefaultColors, DefaultThemeObject } from './theme';
 import { Spinner } from './theme/icons';
 
-const ButtonElement = styled('button')`
-  ${({ theme }: { theme: typeof defaultTheme }) => css`
+const ButtonElement = styled('button', {
```

Comment:
Button is the most noticeable change, especially because the style guide introduces a variant